onnxruntime/core/optimizer/graph_transformer_utils.cc

```diff
@@ -47,6 +48,7 @@ std::vector<std::unique_ptr<RewriteRule>> GenerateRewriteRules(TransformerLevel
       rules.push_back(onnxruntime::make_unique<EliminateSlice>());
       rules.push_back(onnxruntime::make_unique<UnsqueezeElimination>());
       rules.push_back(onnxruntime::make_unique<EliminateDropout>());
+      rules.push_back(onnxruntime::make_unique<ExpandElimination>());
```

Comment:
shall we keep the name aligned with EliminateIdentity/EliminateSlice

Reply:
I think class name is better to be noun instead of verb. Before this, UnsqueezeElimination also uses this format, as well as other types of transformers such as Fusion, Folding, etc.